www/includes/easyparliament/templates/html/alert/index.php

```diff
@@ -0,0 +1,371 @@
+                <p>
+                    You should receive an email shortly which will contain a link.
+                    You will need to follow that link to confirm your email address
+                    and receive future alerts. Thanks.
+                </p>
+
+              <?php } elseif ( $results == 'alert-exists' ) { ?>
+                <h3>You&rsquo;re already subscribed to that!</h3>
+                <p>
+                    It&rsquo;s good to know you&rsquo;re keen though.
+                </p>
+
+              <?php } elseif ( $results == 'alert-already-signed' ) { ?>
+                <h3>We&rsquo;re nearly done</h3>
+                <p>
+                    You should receive an email shortly which will contain a link.
+                    You will need to follow that link to confirm your email address to receive the alert. Thanks.
+                </p>
+
+              <?php } elseif ( $results == 'alert-fail' ) { ?>
+                <h3>Alert could not be created</h3>
+                <p>
+                    Sorry, we were unable to create that alert. Please
+                    <a href="mailto:<?= str_replace('@', '&#64;', CONTACTEMAIL) ?>">let us know</a>.
+                    Thanks.
+                </p>
+
+              <?php } ?>
+            </div>
+        </div>
+      <?php } ?>
+
+      <?php
+          if(
+              (isset($members) && $members->rows() > 0) ||
+              (isset($constituencies) && count($constituencies) > 0) ||
+              ($alertsearch)
+          ) {
+              /* We need to disambiguate the user's instructions */
+      ?>
+        <div class="alert-section alert-section--disambiguation">
+            <div class="alert-section__primary">
+
+              <?php if (isset($members) && $members->rows() > 0) { ?>
+                <h3>Sign up for alerts when people matching <i><?= _htmlspecialchars($alertsearch) ?></i> speak</h3>
+                <ul>
+                  <?php
+                    $q = $members;
+                    for ($n=0; $n<$q->rows(); $n++) {
+                  ?>
+                    <li>
+                        <form action="<?= $actionurl ?>" method="post">
+                            <input type="hidden" name="t" value="<?= _htmlspecialchars($token) ?>">
+                            <input type="hidden" name="email" value="<?= _htmlspecialchars($email) ?>">
+                            <input type="hidden" name="pid" value="<?= $q->field($n, 'person_id') ?>">
+                            When
+                            <?= member_full_name($q->field($n, 'house'), $q->field($n, 'title'), $q->field($n, 'given_name'), $q->field($n, 'family_name'), $q->field($n, 'lordofname') ) ?>
+                            <?php if ($q->field($n, 'constituency')) { ?>
+                                ( <?= $q->field($n, 'constituency') ?> )
+                            <?php } ?>
+                            speaks.
```

Comment:
Can we think of something like 'speaks' that covers the fact lots of it isn't spoken? :)